### PR TITLE
Feature/enhance home and blog

### PR DESCRIPTION
**Pull request by cveneziani**

### CODE REVIEW

File: source/assets/stylesheets/components/_buttons.scss [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
+  }
+
   &.ghost {
-    border: 3px solid white;
+    border: 3px solid $white-pure;
     background-color: transparent;
     letter-spacing: 5px;
     text-transform: uppercase;
+
+    &:hover {
```
Comment (flexbox) [inline source/assets/stylesheets/components/_buttons.scss]: Don't foget to add 

```
&:focus { ... }
```

File: source/blog/index.html.slim [hunk 2]
```diff
@@ -9,8 +9,15 @@ title: Travel stories
...
     - page_articles.each do |article|
-      h2 = link_to article.title, article
-      time datetime="#{article.date}" pubdate=""
-        = article.date.strftime('%m/%d/%Y')
-      p = article.summary
-      = link_to 'Read more →', article
+      div class="row"
+        div class="small-12 medium-3 columns l-blog-cover"
+          - if article.data['cover_picture']
+            = image_tag "#{article.eponymous_directory_path}#{article.data['cover_picture']}"
```
Comment (flexbox) [inline source/blog/index.html.slim]: Don't forget to add `alt:` for all images

File: source/home/_hero.html.slim [hunk 3]
```diff
@@ -5,8 +5,16 @@ div class="l-hero"
...
       h2 = data.site.baseline
+      p class="sub-baseline"
+        ' Trip planner platform for short-term digital workers
+        span class="info round label" SOON!
       p = link_to "Discover the last rides", "/blog", class: "button ghost"
-      p = link_to 'Learn more', '#pitch', class: "learn-more-link"
+  div class="row"
+    div class="large-12 columns"
+      p
+        = link_to '#pitch', class: "learn-more-link" do
```
Comment (flexbox) [inline source/home/_hero.html.slim]: single quote :pray:

**Comment by flexbox** [comment]
Review done

**Comment by cveneziani** [comment]
Fix done.

**Pull request merged**